server/app/models/barman.js

```diff
@@ -88,7 +87,7 @@ const BarmanSchema = Joi.object({
   firstName: Joi.string(),
   lastName: Joi.string(),
   connection: ConnectionInformationSchema,
-  active: Joi.boolean(),
+  leaveAt: Joi.string().isoDate(),
```

Comment:
I prefer using the joi's date validator `Joi.date()`

Reply:
I use the same validator as dateOfBirth field.